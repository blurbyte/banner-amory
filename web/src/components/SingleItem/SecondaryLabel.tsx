// smaller label below main one

import * as React from 'react';
import styled from 'styled-components';

const SmallText = styled.small`
  font-size: 1rem;
`;

type SecondaryLabelProps = {
  secondaryLabel: string;
};

class SecondaryLabel extends React.Component<SecondaryLabelProps> {
  render() {
    const { secondaryLabel } = this.props;
    return (
      <>
        <br />
        <SmallText>{secondaryLabel}</SmallText>
      </>
    );
  }
}

export default SecondaryLabel;
