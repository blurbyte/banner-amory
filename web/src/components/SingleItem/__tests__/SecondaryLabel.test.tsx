import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import SecondaryLabel from '../SecondaryLabel';

test('renders correctly', () => {
  const component = renderer.create(<SecondaryLabel secondaryLabel="taylor swift" />);
  expect(component.toJSON()).toMatchSnapshot();
});
