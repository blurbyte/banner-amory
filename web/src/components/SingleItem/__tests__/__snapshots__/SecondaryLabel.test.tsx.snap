// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <br />,
  .c0 {
  font-size: 1rem;
}

<small
    className="c0"
  >
    taylor swift
  </small>,
]
`;
